### ISSUE TITLE
Question about the policy loss calculation?

### ISSUE THREAD

**Issue by p-morais** [ikostrikov/pytorch-a3c#31]
On line 103 of train.py you calculate the policy loss as 
```python
policy_loss = policy_loss - log_probs[i] * Variable(gae) - 0.01 * entropies[i]
```

~~My question is where does this `-0.01 * entropies[i]` term come from and what does it do?~~

EDIT: I went through the paper again and now see this was a form of regularization they used in the experimental setup section that wasn't in the pseudocode. Thanks for the code!
